about/release-notes.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Timescale release notes and future plans
+excerpt: New features and fixes are released regularly in all Timescale products
+keywords: [TimescaleDB, Timescale Cloud, TimescaleDB Toolkit, pgai, pgvectorscale, pgspot, live-migration, upgrades, updates, releases]
+---
+
+# Timescale release notes
+
+For information about major release for all Timescale product, see the [Changelog][changelog]. For product release 
+notes, see:
+
+* [TimescaleDB](https://github.com/timescale/timescaledb/releases) -  an open-source database that makes SQL scalable for time-series data.
+* [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit/releases) - ease all things analytics when using TimescaleDB.
+* [pgai](https://github.com/timescale/pgai/releases) - brings AI workflows to your PostgreSQL database.
+* [pgvectorscale](https://github.com/timescale/pgvectorscale/releases/tag/0.2.0) -  higher performance embedding search and cost-efficient storage for AI applications.
+* [pgspot](https://github.com/timescale/pgspot/releases) - spot vulnerabilities in PostgreSQL extension scripts.
+* [live-migration](https://hub.docker.com/r/timescale/live-migration/tags) - a Docker image to migrate data to a Timescale Cloud service.
```

Comment:
These are not release notes. We should probably have release notes for live-migration somewhere. @VineethReddy02 are we keeping release notes for this project in any public location?